src/routes/readCompanies.ts

```diff
@@ -32,15 +32,107 @@ function isNumber(n: unknown): n is number {
   return Number.isFinite(n)
 }
 
-const origins =
-  process.env.NODE_ENV === 'development'
-    ? ['http://localhost:4321']
-    : ['https://beta.klimatkollen.se', 'https://klimatkollen.se']
+import apiConfig from '../config/api'
 
-router.use(enableCors(origins))
+router.use(enableCors(apiConfig.frontendURL))
```

Comment:
This needs to be fixed:

```suggestion
router.use(enableCors(apiConfig.origins))
```

Also, the `developmentOrigins` and `productionOrigins`, defined in `constants` are not used.